src/app/components/event/report/EventReportDrawer.vue

```diff
@@ -115,10 +115,16 @@ const isOpen = defineModel<boolean>('open')
 const { step } = useStepper<'reportConfirmation' | 'blockConfirmation'>()
 const onAnimationEnd = (isOpen: boolean) => {
   if (isOpen) return
+
+  if (wasOrganizerBlocked.value) {
+    backToDashboard()
+    return
+  }
```

Comment:
I think you can just check if `step === 'blockConfirmation'`, no? Then you don't need to track a new `wasOrganizerBlocked`.